src/services/CertificateGenerationService.ts

```diff
@@ -245,6 +245,9 @@ class CertificateGenerationService {
                     TankCode: this.containsTankDetails(adrDetails) ? adrDetails.tank.tankDetails.tankCode : undefined,
                     SpecialProvisions: this.containsTankDetails(adrDetails) ? adrDetails.tank.tankDetails.specialProvisions : undefined,
                     TankStatement: (adrDetails && adrDetails.tank) ? adrDetails.tank.tankStatement : undefined,
+                    AdrCertificateNotes: (adrDetails) ? adrDetails.adrCertificateNotes : undefined,
```

Comment:
Something very minor but I think we could perhaps use short-circuit operator or chaining operator to make it more concise.
What do you think?

Reply:
Requested change completed. I used the short-circuit operator.